api/base/serializers.py

```diff
@@ -20,12 +20,12 @@
 from api.base.exceptions import TargetNotSupportedError
 from api.base.exceptions import RelationshipPostMakesNoChanges
 from api.base.settings import BULK_SETTINGS
-from api.base.utils import absolute_reverse, extend_querystring_params, get_user_auth, extend_querystring_if_key_exists
+from api.base.utils import absolute_reverse, extend_querystring_params, get_user_auth, extend_querystring_if_key_exists, rapply
```

Comment:
Possibly out of scope, but since this PR is touching so many imports anyways, it might be a nice opportunity to clean up our imports.
```
from api.base import utils
from osf.utils import permissions
from osf.utils import sanitize
```
instead of 
```
from api.base.utils import absolute_reverse, extend_querystring_params, get_user_auth, extend_querystring_if_key_exists, rapply
from osf.utils.sanitize import strip_html
from osf.utils.permissions import check_private_key_for_anonymized_link
```

https://cosdev.readthedocs.io/en/latest/style_guides/python.html#imports